management/management_request.go

```diff
@@ -156,10 +160,10 @@ func (o *requestOption) apply(r *http.Request) {
 func applyListDefaults(options []RequestOption) RequestOption {
 	return newRequestOption(func(r *http.Request) {
 		PerPage(50).apply(r)
+		IncludeTotals(true).apply(r)
```

Comment:
Nice 👍🏻 this now allows us to also set include totals to false if needed, otherwise it would've been impossible to do so.